CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Removed
```

Comment:
I think it is more of a 'Fixed' rather than 'Removed', since you are fixing a bug here